DEVELOPMENT.md

```diff
@@ -0,0 +1,64 @@
+
+## One Time Setup
+
+Install goose for db migrations:
+
+```
+$ go install github.com/pressly/goose/v3/cmd/goose@latest
+```
+
+## Create and Populate PostgreSQL Database
+
+Launch postgresql: 
+
+```bash
+podman run --name sippy-postgres -e POSTGRES_PASSWORD=password -p 5432:5432 -d quay.io/enterprisedb/postgresql
+```
+
+TODO: podman run postgres as a regular user may require disabling selinux on Fedora 35. Needs investigation.
+
+## Populating Data
+
+### From Scratch
+
+Populate db schema with goose, may need to run periodically if new migrations come in, fetch data from testgrid, and load it into the db. This process can take some time for recent OpenShift releases with substantial number of jobs. Using older releases like 4.7 can be fetched and loaded in just a couple minutes.
+
+```bash
+goose --dir dbmigration postgres "user=postgres password=password dbname=postgres sslmode=disable" up
```

Comment:
Goose supports doing this inline with embedded migrations: https://github.com/pressly/goose#embedded-sql-migrations

Is the standalone CLI command better for some reason?

Reply:
Hmm that looks interesting. CLI does give us admin level control to work with the migrations, but we could probably break glass to do that anyhow even if we embed and manage the schema automatically.

Is it ok if I take this as an action item for a next pass? I'm in a pickle with landing the kube work which is based on this ASAP and a bit underwater with watcher stuff on top. I will file a jira.